### PR TITLE
Release 0.14.0

### PR DESCRIPTION
**Pull request by mli**
**Merging this PR means we will update both d2l.ai and upload d2l to pypi**

It contains all changes since 0.8.0, mainly pytorch/tensorflow porting for the first 8 chapters.

We bumped from 0.8.0 to 0.14.0 to align the version numbers of the notebooks (0.8) and d2l package (0.13.2). This misalignment is the main reason causes error for a user to run a notebook.

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

File: config.ini [hunk 1]
```diff
@@ -69,15 +77,17 @@ mono_font = Inconsolata
...
+
+version = 0.13.2
 
-# The parttern to mark this block will be saved.
-save_mark = Saved in the d2l package for later use
+version_file = d2l/__init__.py
 
 
 [deploy]
-s3_bucket = s3://en.d2l.ai
```
Comment (astonzhang) [inline config.ini:80]: Do we want to keep it when merging?

Comment (mli) [reply config.ini:80]: it's not needed anymore, and it's not safe if others copy it

**Comment by mli** [comment]
Job d2l-en/PR-1092/8 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1092/

**Comment by mli** [comment]
Job d2l-en/PR-1092/9 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1092/

**Comment by mli** [comment]
Job d2l-en/PR-1092/10 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1092/

**Comment by mli** [comment]
Job d2l-en/PR-1092/12 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1092/

**Pull request merged**